src/Enterspeed.Cli/Commands/Schema/ImportSchemaCommand.cs

```diff
@@ -0,0 +1,138 @@
+﻿using System.CommandLine;
+using System.CommandLine.Invocation;
+using System.Text.Json;
+using Enterspeed.Cli.Api.MappingSchema;
+using Enterspeed.Cli.Services.ConsoleOutput;
+using Enterspeed.Cli.Services.FileService;
+using Enterspeed.Cli.Services.FileService.Models;
+using MediatR;
+using Microsoft.Extensions.Logging;
+
+namespace Enterspeed.Cli.Commands.Schema;
+
+internal class ImportSchemaCommand : Command
+{
+    public ImportSchemaCommand() : base(name: "import", "Imports all schemas from the /schemas folder on the disk. Will create new schemas and update existing schemas if --override is enabled.")
+    {
+        AddOption(new Option<bool>(new[] { "--override", "-o" }, "Override existing schemas"));
+    }
+
+    public new class Handler : BaseCommandHandler, ICommandHandler
+    {
+        private readonly IMediator _mediator;
+        private readonly IOutputService _outputService;
+        private readonly ISchemaFileService _schemaFileService;
+        private readonly ILogger<CreateSchemaCommand> _logger;
+
+        public Handler(IMediator mediator, 
+            IOutputService outputService, 
+            ISchemaFileService schemaFileService, 
+            ILogger<CreateSchemaCommand> logger)
+        {
+            _mediator = mediator;
+            _outputService = outputService;
+            _schemaFileService = schemaFileService;
+            _logger = logger;
+        }
+
+        public bool Override { get; set; }
+
+        public async Task<int> InvokeAsync(InvocationContext context)
+        {
+            var existingSchemas = await _mediator.Send(new QueryMappingSchemasRequest());
+            var schemaFiles = _schemaFileService.GetAllSchemas();
+
+            var successfulImports = 0;
+            var failedImports = 0;
+            foreach (var schemaFile in schemaFiles)
+            {
+                var matchingSchema = existingSchemas.FirstOrDefault(sc => sc.ViewHandle == schemaFile.Alias);
+                var schemaExists = matchingSchema is not null;
+
+                if (schemaExists && Override)
+                {
+                    if (await UpdateExistingSchema(schemaFile, matchingSchema))
+                        successfulImports += 1;
+                    else
+                        failedImports += 1;
+                }
+                else if (!schemaExists)
+                {
+                    if (await CreateNewSchema(schemaFile))
+                        successfulImports += 1;
+                    else
+                        failedImports += 1;
+                }
+            }
+
+            _outputService.Write($"Successfully imported {successfulImports} schema(s).");
+                
+            if (failedImports > 0)
+            {
+                _outputService.Write($"Failed to imported {failedImports} schema(s).");
+            }
```

Comment:
We should indicate that somthing failed, with a returncode of != 0

Reply:
Done